assets/src/edit-story/app/story/useStoryReducer/reducers/updateElements.js

```diff
@@ -70,16 +70,51 @@ function updateElements(
     return state;
   }
 
-  const updatedElements = oldPage.elements.map((element) => {
-    if (!idsToUpdate.includes(element.id)) {
-      return element;
+  const {
+    animationLookup,
+    elements: updatedElements,
+  } = oldPage.elements.reduce(
+    ({ animationLookup, elements }, element) => {
+      if (!idsToUpdate.includes(element.id)) {
+        return {
+          animationLookup,
+          elements: [...elements, element],
+        };
+      }
+
+      const { animation, ...elem } = updateElementWithUpdater(
+        element,
+        propertiesOrUpdater,
+        pageIndex
+      );
+
+      const animLookup = animation
+        ? { [animation.id]: { ...animation, targets: [elem.id] } }
```

Comment:
Are we worried that we might perform this operation on  an existing animation that has more than 1 id and it would just drop all the other ids?

Reply:
I don't think that will be a case in newly created things, but just existing template stuff

Reply:
Nope, multiple targets is a feature that we didn't need to implement, it'll just be one to one 😱